.github/workflows/test-integration.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Test Integration"
+
+on:
+  workflow_run:
+    workflow: [Release]
```

Comment:
The `Release` workflow runs only on [push](https://github.com/FuelLabs/fuels-ts/blob/dc17b9c454fb025cd07840e79489afe346ff6114/.github/workflows/release.yaml#L3) to `master`/`release/*` branches, while in #2170 
> The idea is that every PR would publish a PR tag and have an install/assertion routine.

This would mean that we'd catch* regressions only _after_ merge to `master`. Am I missing something here?

*and only if we go to the actions tab and find that the workflow failed.

Reply:
Can't we just use `needs`?

https://github.com/FuelLabs/fuels-ts/blob/eefe0079eb253fcbf94d09a26c102d35366c696d/.github/workflows/test.yaml#L104

Reply:
Oh, I see; we are talking cross-workflows. I'm not sure, maybe not.

Reply:
That's perfect! 💯 

Feel free to resolve this conversation when it's done. (not sure if it is already)